reflekt-plugin/src/main/kotlin/io/reflekt/plugin/ReflektComponentRegistrar.kt

```diff
@@ -42,7 +42,8 @@ class ReflektComponentRegistrar(private val hasConfiguration: Boolean = true) :
         val outputDir = configuration[Keys.OUTPUT_DIR]
         val reflektContext = ReflektContext()
 
-        // Todo: will this be called multiple times (for each project module)? can we avoid this?
+        // Will this be called multiple times (for each project module),
```

Comment:
Sorry, I'm confused, is it a question or a statement? Could you please either add a question mark or change the order of the words?